src/store/reducers/from_stix.js

```diff
@@ -22,16 +22,18 @@ const FromSTIXReducer = (state = INITIAL_STATE, action) => {
   switch (action.type) {
     case ADD_FIELD: {
       const field = action.payload.field;
+      const currentStixMapping = state.stixMapping;
       if (!(field in state.stixMapping)) {
         return {
           ...state,
           stixMapping: {
-            ...state.stixMapping,
             [field]: {
               values: [],
             },
+            ...currentStixMapping
```

Comment:
JS objects are unordered /  it does not (always) follow the insertion order. Check if `Map` object is better suited to solve the issue
https://stackoverflow.com/a/5525820